connectors/airwatch/src/main/resources/static/discovery/metadata.json

```diff
@@ -1,16 +1,28 @@
 {
   "backend_base_url_header": "x-airwatch-base-url",
-  "object_type": "card",
-  "fields": {
-    "udid": {
-      "env": "ENROLLMENT_DEVICE_UDID"
-    },
-    "platform": {
-      "env": "DEVICE_SYSTEM_NAME"
-    },
-    "app_keywords": {
-      "capture_group": 1,
-      "regex": "(CONNECTOR_REGEX)"
+  "object_types": [
+    {
+      "object_type": {
+        "name": "card",
+        "doc": {
+          "href": "https://github.com/vmwaresamples/card-connectors-guide/wiki/Card-Responses"
+        }
+      },
+      "fields": {
+        "udid": {
+          "env": "ENROLLMENT_DEVICE_UDID"
+        },
+        "platform": {
+          "env": "DEVICE_SYSTEM_NAME"
+        },
+        "app_keywords": {
+          "capture_group": 1,
+          "regex": "(CONNECTOR_REGEX)"
+        }
+      },
+      "endpoint": {
+        "href": "CONNECTOR_HOST/cards/requests"
```

Comment:
This should end up `"https://my-connector/cards/requests"` because of the X-Forwarded headers set in ControllerTestsBase: https://github.com/vmware/connectors-workspace-one/blob/7c6bb345131f8a3c1e64eb67333a4cda90d26ffd/common/connectors-test/src/main/java/com/vmware/connectors/test/ControllerTestsBase.java#L144